packages/hexboosting-sdk/src/pdas.ts

```diff
@@ -2,15 +2,30 @@ import { PublicKey } from "@solana/web3.js";
 import { PROGRAM_ID } from "./constants";
 import BN from "bn.js";
 
+const deviceTypeValues = {
```

Comment:
Could be a typescript enum? Might be able to typecheck against the DeviceTypeV0 too not sure

```javascript
enum DeviceType {
    cbrsIndoor,
    cbrsOutdoor,
    wifiIndoor,
    wifiOutdoor,
}

console.log(DeviceType[DeviceType['cbrsIndoor']]); // => "cbrsIndoor"
console.log(DeviceType[0]) // => "cbrsIndoor"
console.log(DeviceType.cbrsIndoor) // => 0
```